src/main/scala/io/iohk/ethereum/consensus/pow/EthashDAGManager.scala

```diff
@@ -13,11 +13,11 @@ class EthashDAGManager(blockCreator: EthashBlockCreator) extends Logger {
   var currentEpochDagSize: Option[Long] = None
   var currentEpochDag: Option[Array[Array[Int]]] = None
 
-  def calculateDagSize(blockNumber: Long, epoch: Long): (Array[Array[Int]], Long) = {
+  def calculateDagSize(blockNumber: Long, epoch: Long, ecip1099ActivationBlock: Long): (Array[Array[Int]], Long) = {
```

Comment:
wouldn't it make sense to make this parameter a characteristic of the manager itself? this cannot change during the run, and passing it all around seems tedious.